src/Build.UnitTests/EscapingInProjects_Tests.cs

```diff
@@ -715,14 +716,14 @@ public void EscapedWildcardsShouldNotBeExpanded()
         [Trait("Category", "mono-osx-failing")]
         public void EscapedWildcardsShouldNotBeExpanded_InTaskHost()
         {
-            MockLogger logger = new MockLogger();
+            MockLogger logger = new();
 
             try
             {
                 // Populate the project directory with three physical files on disk -- a.weirdo, b.weirdo, c.weirdo.
                 EscapingInProjectsHelper.CreateThreeWeirdoFiles();
                 Project project = ObjectModelHelpers.CreateInMemoryProject(@"
-                <Project ToolsVersion=`msbuilddefaulttoolsversion` xmlns=`http://schemas.microsoft.com/developer/msbuild/2003`>
```

Comment:
I believe so. Every time I've asked about them, the answer has been that it's a red herring because they don't do anything. Since they're still supported, it can be useful to make sure there's at least a few cases somewhere in our code base, but there are currently plenty. Also, I think dfederm has a PR out to remove a lot more.